src/pages/contributing/get-started.mdx

```diff
@@ -66,7 +66,7 @@ Accessibility<br /><br /> **Amaya Mali**<br /> Gantt chart
 <Column className="one" colSm={2} colMd={2} colLg={3}>
 
 **Tom Roach**<br /> Community support<br /><br />**Eugene To **<br />
-Figma<br /><br /> **Shankar Venkatachalam** <br /> Create React App<br /><br />
+Figma<br /><br /> **Shankar Venkatachalam** <br /> React App<br /><br />
```

Comment:
"Create React App" is the correct name I believe https://create-react-app.dev/ (not sure where the Carbon version is)

Reply:
@jeanservaas See alison's comment here. Does this need an update?